### DISCUSSION TITLE
Responsive variable

### DISCUSSION THREAD

**Discussion by imClement** [alpinejs/alpine#1020] [1. Help] [answered]
Hello,

I have a dropdown that needs to start open or start closed depending on the dimension of the window. 
Also, the dropdown should close on resize if the window become less then a certain dimension. 

Is it possible to achieve that with Alpinejs?

Thank you very much for your time,

**Comment by KevinBatdorf** [comment, marked as answer]
Maybe something like this:

```html
<div
  id="dropdown"
  x-data="{ open: false, maybeOpen() { this.open = window.innerWidth > 768 } }"
  @resize.debounce.100.window="maybeOpen()"
  x-show="open"
  x-init="maybeOpen()">
```

**Reply by francoism90** [reply]
@KevinBatdorf  Thanks!

**Comment by imClement** [comment]
Exactly! 

Thank you, @KevinBatdorf, for your time and for your support.

**Comment by skttl** [comment]
Hi @ClementN 

Maybe you can use matchMedia for this?
https://codepen.io/skttl/pen/abmWQbp

**Reply by imClement** [reply]
Thank you, @skttl .
